build.json

```diff
@@ -1,35 +1,35 @@
 {
-  "UseSystemDotNetPath": "false",
-  "DotNetFolder": ".dotnet",
-  "DotNetInstallScriptURL": "https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0-preview2/scripts/obtain",
-  "DotNetChannel": "preview",
-  "DotNetVersion": "1.0.0-preview2-003131",
-  "BuildToolsFolder": ".tools",
-  "ArtifactsFolder": "artifacts",
-  "TestProjects": {
-    "OmniSharp.MSBuild.Tests": [
-      "netcoreapp1.0"
+    "UseSystemDotNetPath": "false",
+    "DotNetFolder": ".dotnet",
+    "DotNetInstallScriptURL": "https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0-preview2/scripts/obtain",
+    "DotNetChannel": "preview",
+    "DotNetVersion": "1.0.0-preview2-003131",
+    "BuildToolsFolder": ".tools",
+    "ArtifactsFolder": "artifacts",
+    "TestProjects": {
+        "OmniSharp.MSBuild.Tests": [
+            "netcoreapp1.0"
+        ],
+        "OmniSharp.Roslyn.CSharp.Tests": [
+            "netcoreapp1.0"
```

Comment:
That's the one. We're still working on getting Roslyn updated to .NET Core 1.1, which will address the problem.